draft-packages/guidance-block/docs/GuidanceBlock.stories.tsx

```diff
@@ -81,6 +81,16 @@ const WithoutActionArrowButton = () => (
   </div>
 )
 
+const WithoutMaxWidth = () => (
+  <div style={{ display: "flex", justifyContent: "center", margin: "100px" }}>
+    <GuidanceBlock
+      img={{ src: guidanceBlockImg, alt: "Guidance block" }}
```

Comment:
nit/ And not that relevant for this repo - but an empty `alt` tag is useful for screen readers when an image is purely decorative / does not add additional context (like this one): https://www.w3.org/WAI/tutorials/images/decorative/

Reply:
I guess its more of a consideration when the component is actually used in the code base, will make sure to add empty string there!